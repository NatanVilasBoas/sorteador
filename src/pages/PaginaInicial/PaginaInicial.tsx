import Formulario from "../../components/Formulario/Formulario"
import ListaParticipantes from "../../components/ListaParticipantes/ListaParticipantes"
import Rodape from "../../components/Rodape/Rodape"

const PaginaInicial = () => {
    return(
        <>
            <Formulario/>
            <ListaParticipantes/>
            <Rodape/>
        </>
    )
}

export default PaginaInicial;