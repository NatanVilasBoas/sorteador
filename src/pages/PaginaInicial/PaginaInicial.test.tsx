import { render } from "@testing-library/react"
import { RecoilRoot } from "recoil"
import PaginaInicial from "./PaginaInicial"

const mockNavegacao = jest.fn();

jest.mock('react-router-dom', () => {
    return{
        useNavigate: () => mockNavegacao
    }
})

describe('a pagina inicial', () => {
    test('deve ser renderizado corretamente', () => {
        const {container} = render(<RecoilRoot>
            <PaginaInicial />
        </RecoilRoot>)

        expect(container).toMatchSnapshot();
    })
})