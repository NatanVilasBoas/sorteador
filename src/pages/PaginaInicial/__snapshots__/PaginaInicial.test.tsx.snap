// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`a pagina inicial deve ser renderizado corretamente 1`] = `
<div>
  <form>
    <input
      placeholder="insira os nomes dos participantes"
      type="text"
      value=""
    />
    <button
      disabled=""
    >
      Adicionar
    </button>
    
  </form>
  <ul />
  <footer>
    <button
      disabled=""
    >
      Iniciar Brincadeira
    </button>
  </footer>
</div>
`;
